packages/react-reconciler/src/__tests__/ReactSuspense-test.property.js

```diff
@@ -0,0 +1,281 @@
+            // It may contain successes and rejections
+            const cache = new Map();
+            const readOrThrow = text => {
+              if (cache.has(text)) {
+                // The text has already been queried
+                const {promise, resolvedWith} = cache.get(text);
+                // Not resolved yet?
+                if (resolvedWith === null) throw promise;
+                // Success
+                return text;
+              } else {
+                // Not yet queried
+                const promise = s.schedule(
+                  Promise.resolve(),
+                  `Resolve async request for text ${JSON.stringify(text)}`,
+                  {type: 'request', text},
+                );
+                const cachedValue = {promise, resolvedWith: null};
+                promise.then(success => (cachedValue.resolvedWith = {success}));
+                cache.set(text, cachedValue);
+                throw promise;
+              }
+            };
+
+            let setText;
+            function App() {
+              const [text, _setText] = React.useState(initialText);
+              setText = _setText;
+              return <AsyncText text={text} readOrThrow={readOrThrow} />;
+            }
+
+            // Initial render
+            ReactNoop.render(
+              <Suspense fallback={<Text text="Loading..." />}>
+                <App />
+              </Suspense>,
+            );
+            flushAndYieldScheduler();
+            expect(ReactNoop).toMatchRenderedOutput(<span>Loading...</span>);
+
+            // Schedule updates into the scheduler
+            // Updates will not be reordered
+            // BUT promises that they may trigger may be scheduled in-between
+            s.scheduleSequence(
+              textUpdates.map(update => {
+                return {
+                  label: `Call setText(${JSON.stringify(
+                    update.text,
+                  )}) with priority ${update.priority}`,
+                  metadata: {type: 'setText', ...update},
+                  builder: async () =>
+                    Scheduler.unstable_runWithPriority(update.priority, () => {
+                      setText(update.text);
+                    }),
+                };
+              }),
+            );
+
+            // Exhaust the queue of scheduled tasks
+            while (s.count() !== 0) {
+              await ReactNoop.act(async () => {
+                await s.waitOne();
+                flushAndYieldScheduler();
+              });
+            }
+
+            // Check the final value is the expected one
+            const lastText =
+              textUpdates.length > 0
+                ? textUpdates[textUpdates.length - 1].text
+                : initialText;
+            expect(ReactNoop).toMatchRenderedOutput(<span>{lastText}</span>);
+          },
+        )
+        .beforeEach(beforeEachAction),
+      {
+        examples: [
```

Comment:
You can directly copy-paste the reported errors to run them everytime you will execute this test in the future.

See:
```
         [
            fc.schedulerFor()`
        -> [task${2}] sequence::Call setText("1") with priority 1 resolved
        -> [task${1}] promise::Resolve async request for text "" resolved`,
            '',
            [{priority: 1, text: '1'}],
          ]
```